pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -90,12 +92,19 @@ buildNpmPackage {
     makeBinaryWrapper
     rdfind
     unzip
+    bash
   ];
 
   patches = [
     # fix httpSafePort setting
     # https://github.com/cryptpad/cryptpad/pull/1571
     ./0001-env.js-fix-httpSafePort-handling.patch
+    # https://github.com/cryptpad/cryptpad/pull/1740
+    (fetchpatch {
+      name = "Add a `--check-only` command to `install-onlyoffice.sh`";
+      url = "https://patch-diff.githubusercontent.com/raw/cryptpad/cryptpad/pull/1740.patch";
```

Comment:
This isn't stable if you update the PR so perhaps make it a link to the commit itself or just include it here as a separate file like I did for the other one?

(If I had any hope it would get merged quickly I wouldn't mind, but I haven't had much luck in the past with contributing to cryptpad, I wish you more success! It's great to strive to upstream things like this.)

Reply:
I've updated this to be a permalink, as you suggest. I *think* that's preferable to try to keep the size of nixpkgs down?

> If I had any hope it would get merged quickly I wouldn't mind, but I haven't had much luck in the past with contributing to cryptpad, I wish you more success! It's great to strive to upstream things like this

That's a bummer to hear. :crossed_fingers: